.travis.yml

```diff
@@ -143,6 +143,24 @@ jobs:
         - scripts/travis-helmtest.sh v1.18.5 --test-cephfs=false
           --test-rbd=true || travis_terminate 1;
 
+    - stage: upgrade testing
+      name: CephFS upgrade tesing with CSI v2.0.0
+      script:
+        - scripts/skip-doc-change.sh || travis_terminate 0;
+        - make image-cephcsi || travis_terminate 1;
+        - scripts/travis-functest.sh v1.18.5 --test-cephfs=true
+          --test-rbd=false -upgrade-testing=true
+          -upgrade-version=v2.0.0 || travis_terminate 1;
```

Comment:
the flag should start with `--` double hyphen . and also can you please update the version to the latest released version 2.1.2